CHANGELOG.next.asciidoc

```diff
@@ -54,6 +54,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Affecting all Beats*
 
+- Fix a yaml typo in libbeat/common/transport/tlscommon/config.go {issue}10871[10871], {pull}12354[12354]
```

Comment:
I think one line should be enough to describe the change. As the Changelog is intended for users, mentioning `tls.Config` is not appropiate. I would reword this.

Reply:
@kvch your proposition sounds and looks better, thank you! I updated the entry accordingly!